ebos/eclbasevanguard.hh

```diff
@@ -235,10 +245,14 @@ public:
 
         EWOMS_REGISTER_PARAM(TypeTag, bool, OwnerCellsFirst,
                              "Order cells owned by rank before ghost/overlap cells.");
+#if HAVE_MPI
```

Comment:
Can you have a look at this please, @blattms?   I seem to recall that there is at least **one** workflow/use case that wants to leverage Zoltan support even if we don't have MPI.  OpenCL and/or Cuda, possibly?

Reply:
i was actually looking into that. as is, there is no calls to that code, with or without mpi, i figured we can quell the options for now. but maybe that would break some unmerged code?

Reply:
that being said, as we don't even link zoltan without mpi we would have more things to cover in that case.

Reply:
aha, there's the call i could not findwhile grepping. but yeah, it's still under HAVE_MPI, https://github.com/OPM/opm-grid/blob/master/opm/grid/common/ZoltanPartition.hpp#L90